packages/service-library/src/servicelib/fastapi/requests_decorators.py

```diff
@@ -119,4 +119,4 @@ async def wrapper(request: Request, *args, **kwargs):
     return wrapper
 
 
-__all__: tuple[str, ...] = ("cancel_on_disconnect",)
+__all__: tuple[str, ...] = "cancel_on_disconnect"
```

Comment:
this should be a tuple, I think this change might be wrong?

Reply:
Right, this is a mistake. I will fix